ideascube/library/static/library/library.css

```diff
@@ -39,4 +39,145 @@ img.cover {
 .book-detail h3 {
     font-family: 'fira_sansmedium';
     font-size: 1em;
-}
\ No newline at end of file
+}
+
+
+
+/* ************************************************* */
+/* ************ SECTION COLORS ********************* */
+/* ************************************************* */
+
+/************* Cartoons *************/
+#id_section > option[value=children-cartoons] {
```

Comment:
I'd rather put a small square of the given color close to the option text, it's a bit flashy as it is ;)